test/fit_tests/common/fit_common.py

```diff
@@ -363,7 +363,7 @@ def restful(url_command, rest_action='get', rest_payload=[], rest_timeout=None,
                                        verify=sslverify,
                                        )
         if rest_action == "binary-put":
-            rest_headers.update({"Content-Type": "application/octet-stream"})
+            rest_headers.update({"Content-Type": "application/x-www-form-urlencoded"})
```

Comment:
Content-type has been a problem in the binary put and I don't know if this is a good universal solution for it. This seems to work for all of the cases that I have tested.